### DISCUSSION TITLE
How to achieve graph as in Obsidian?

### DISCUSSION THREAD

**Discussion by indicozy** [graphology/graphology#490] [Q&A]
Hi there!
I'm trying to replicate the graph rendered in Obsidian:
![image](https://github.com/graphology/graphology/assets/51142471/ea19ea27-7056-4b1c-8bd5-01097287f4b5)

But I have: 
![image](https://github.com/graphology/graphology/assets/51142471/0f5065e8-be62-4999-8350-1204e6f43bd5)

My code: https://github.com/indicozy/website/blob/master/src/components/obsidian/ObsidianGraph.tsx
I copied my code from here: https://codesandbox.io/s/github/jacomyal/sigma.js/tree/main/examples/mouse-manipulations?file=/index.ts

What am I doing wrong?

**Comment by Yomguithereal** [comment]
@indicozy `graphology` is not a rendering library. You are confusing it with `sigma`, and you should probably enquire there. `graphology` is just a graph data structure and has nothing to do with the problem you are having sorry. This said, I'd say you either have a layout tweaking problem or a data serialization problem here.

**Reply by indicozy** [reply]
I understand that graphology is not a rendering library, yet sigma is relying on it to render graphs. Since physics engine (layout) is working directly with graphology nodes, I believe it is related more to graphology than to sigma.
